src/status_im/translations/en.cljs

```diff
@@ -39,17 +39,17 @@
    :status-console                       "Status Console"
 
    ;;sign in
-   :intro-text                           "Status is an open source decentralized chat and Ethereum browser"
-   :intro-text-description               "Status is built with the help of the community to help you use all the benefits of decentralized web in your mobile phone"
+   :intro-text                           "Status is your gateway to the decentralized web"
+   :intro-text-description               "A community-powered platform to securely chat and transact on the Ethereum blockchain"
    :create-account                       "Create account"
    :already-have-account                 "I already have an account"
-   :creating-your-account                "Creating your account on the blockchain. We can't touch it, no one can, except for you!"
-   :password-placeholder                 "Type your password"
+   :creating-your-account                "Creating your account on the blockchain..."
+   :password-placeholder                 "At least 6 characters"
    :password-placeholder2                "Confirm your password"
    :name-placeholder                     "Display name"
-   :password_error1                      "Password confirmation doesn't match password."
+   :password_error1                      "Password don't match."
```

Comment:
This reads a little funny. Since it is referring to two passwords should it be plural?

Reply:
Typo, thanks